### DISCUSSION TITLE
pip uninstall lightning does not uninstall pl/app, should it?

### DISCUSSION THREAD

**Discussion by awaelchli** [Lightning-AI/pytorch-lightning#14850] [General]
When you install the lightning package with `pip install lightning`, it brings with it pytorch_lightning and the lightning_app packages. Should `pip uninstall lightning` remove these two?

cc @Borda

**Comment by Borda** [comment]
Not really, [pypi version](https://pypi.org/project/lightning/) is a thin meta-package with set requirements for the `pytorch-lightning` and the `lightning-app`...
But if you install from source code via calling `pip install .` or `pip install https://github.com/Lightning-AI/lightning/archive/refs/heads/master.zip`, it will install both mentioned packages... :rabbit: 
So to say, if you install from `pip`, you get this package but install from source, you get a rich package... :otter:
